src/policy/mallocspace/global.rs

```diff
@@ -23,6 +22,7 @@ use std::marker::PhantomData;
 use std::sync::atomic::AtomicU32;
 use std::sync::atomic::{AtomicUsize, Ordering};
 // only used for debugging
+use crate::util::malloc::library::{BYTES_IN_MALLOC_PAGE, LOG_BYTES_IN_MALLOC_PAGE};
```

Comment:
This should be moved above the comment as it currently seems like the import is "only used for debugging" which is obviously not true